src/installer.ts

```diff
@@ -86,3 +87,62 @@ export async function installPlaywright(vscode: vscodeTypes.VSCode) {
 
   terminal.sendText(`npm init --yes playwright@latest -- --quiet ${args.join(' ')}`, true);
 }
+
+export async function installBrowsers(vscode: vscodeTypes.VSCode, model: TestModel) {
+  const chromium: vscodeTypes.QuickPickItem = {
+    label: 'Chromium',
+    picked: true,
+    description: '— powers Google Chrome, Microsoft Edge, etc\u2026',
+  };
+  const firefox: vscodeTypes.QuickPickItem = {
+    label: 'Firefox',
+    picked: true,
+    description: '— powers Mozilla Firefox',
+  };
+  const webkit: vscodeTypes.QuickPickItem = {
+    label: 'WebKit',
+    picked: true,
+    description: '— powers  Apple Safari',
+  };
+  const installDepsAction: vscodeTypes.QuickPickItem = {
+    label: 'Install Linux dependencies',
+    picked: process.platform === 'linux' && !fs.existsSync(path.join(process.env.XDG_CACHE_HOME || path.join(os.homedir(), '.cache'), 'ms-playwright')),
```

Comment:
why do we need this `!fs.existsSync(path.join(process.env.XDG_CACHE_HOME` ?

Reply:
yes!